src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java

```diff
@@ -104,9 +109,10 @@ public void testLiteralWhile() throws Exception {
                 createCheckConfig(WhitespaceAfterCheck.class);
         configurationTestLiteralWhile.addAttribute("tokens", "LITERAL_WHILE");
         final String[] expected = {
-            "39:14: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "while"),
+            "16:14: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "while"),
```

Comment:
We are not expecting line numbers to change. Files should be like 99% direct copy/move.
Package name and class name can be changed if it is not dependent on test. All others should stay as is.